src/include/common/types/timestamp_t.h

```diff
@@ -63,6 +63,8 @@ class Timestamp {
     // Create a Timestamp object from a specified (date, time) combination.
```

Comment:
Not related to changes in this PR. But I think it makes more sense to shape this file to our code style, for example ,t he function name should start with a lower case char. I don't think it makes sense now to keep them same as DuckDB's, since we are exposing them as our public api and also adding new functions.

Ditto same for date_t.h, which now contains mixed of both styles.

You're welcome to address them in this PR, or if not, can you left a TODO or open another PR for this?

Reply:
Solved in this pr